### ISSUE TITLE
♿ Android - Accessibility - Create event - Event time button touch target not accessible with TalkBack

### ISSUE THREAD

**Issue by ommacc** [CMMS-Esmorga/Esmorga-Project#76]
**Original Jira Issue:** [ESM-433](https://corunamobilemakers.atlassian.net/browse/ESM-433)

## Description

Current behaviour: Using TalkBack, the user is not able to select the outer numbers, only those that are closer to the center

Expected behavior: Buttons should meet minimum touch target size to be accessible for all users

## Metadata

- **Type:** Bug
- **Status:** READY FOR DEV
- **Original Jira created:** 2026-01-26T13:54:20.875+0100
- **Priority:** Medium
- **Reporter:** QA Team
- **Assignee:** Android
- **Labels:** Android

## Attachments

![image-20260126-125608.png](https://raw.githubusercontent.com/CMMS-Esmorga/Esmorga-Project/main/.jira-attachments/ESM-433/10700-image-20260126-125608.png)

## Custom Fields

- **Component:** App - Frontend
- **Environment:** QA
- **Priority:** 🔵 Low
- **Steps to reproduce:** - Activate TalkBack
- Login as an #admin user
- Navigate to “Create an event”
- Fill all the mandatory data until you reach “Date & Time” screen
- Try to change the hour to 0 to 11
- **Rank:** 0|i002lr:
- **Development:** {}

## Related Issues
- **relates to:** #168
